Library/Formula/tarantool.rb

```diff
@@ -19,22 +19,12 @@ def install
     args = std_cmake_args
     args << "-DCMAKE_INSTALL_MANDIR=#{doc}"
     args << "-DCMAKE_INSTALL_SYSCONFDIR=#{etc}"
+    args << "-DCMAKE_INSTALL_LOCALSTATEDIR='#{var}'"
```

Comment:
I'd rather we fix it. Someone will spot the inconsistency and try to fix it either by removing the `''` or adding them across the other options, and I'll have likely forgot this conversation by the time that happens.